### DISCUSSION TITLE
add a timer to a target?

### DISCUSSION THREAD

**Discussion by P-I-Engineer** [iNavFlight/inav#6428] [General]
i got camera control to work on the LED pin on the betaflightF4 target.

I'm trying to get it to work on an RX pin. I'm trying to add a timer for the RX6 pin and it's not working. any suggestions? or am i attacking this incorrectly? I've tried adding additional timers, that didn't work either.

my goal is to use the RX4 pad on the matek f722se mini board for camera control. I'm dev'ing with a betaflightF4 board.

![image](https://user-images.githubusercontent.com/11393479/102420666-85208c80-3fb7-11eb-9216-f1ab4a95ff25.png)

![image](https://user-images.githubusercontent.com/11393479/102420691-8ce03100-3fb7-11eb-96dc-b5ff9c83e5df.png)

**Comment by P-I-Engineer** [comment]
thanks to Pawel's video, i see my error, must use stm F405 Spec to figure out timers.
https://www.youtube.com/watch?v=v4R-pnO4srU

Thanks Pawel

**Comment by zvikaf** [comment]
Did you figured it ?
